core/src/blocktree.rs

```diff
@@ -566,7 +579,9 @@ impl Blocktree {
                     self.erasure_meta_cf
                         .get((blob_slot, set_index))
                         .expect("Expect database get to succeed")
-                        .unwrap_or_else(|| ErasureMeta::new(set_index))
+                        .unwrap_or_else(|| {
+                            ErasureMeta::new(set_index, &erasure_config_opt.unwrap())
```

Comment:
you're certain you'll have an erasure_config_opt by this point?

Reply:
Yes, the code at line 557 ensures that we have a config by this point.